### PR TITLE
refactor: Only make bloc factories internal

### PR DESCRIPTION
**Pull request by provokateurin**
Closes https://github.com/nextcloud/neon/issues/1372

For the neon_* packages this has not effect since they still have the global ignore set (which we should change eventually).

### CODE REVIEW

File: packages/neon_framework/lib/src/blocs/login_check_account.dart [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: public_member_api_docs
```
Comment (Leptopoda) [inline packages/neon_framework/lib/src/blocs/login_check_account.dart:1]: IIRC the constructors had docs before all these changes. Please restore them or add new ones.

**Review by Leptopoda — CHANGES_REQUESTED** [review]
It's not that hard to write propper documentation here and we should do so moving forward.

File: packages/neon_framework/lib/src/blocs/login_check_account.dart (hunk 1 above)
Comment (provokateurin) [reply packages/neon_framework/lib/src/blocs/login_check_account.dart:1]: Nope that was not the case.

Comment (Leptopoda) [reply packages/neon_framework/lib/src/blocs/login_check_account.dart:1]: Then please add docs.

**Comment by provokateurin** [comment]
I agree and I plan to do that, but for now I only want to change this to enable it. I'll create another issue to document all the blocs.

**Comment by provokateurin** [comment]
As the commit says this is only a refactor to enabled easier detection of missing docs. I do not want to add the docs in this PR already.

**Review by Leptopoda — APPROVED** [review]
I still disagree.
It's not a big deal to add a few lines of docs. I fear that any open issue will just rot (similar to https://github.com/nextcloud/neon/issues/1119)

**Pull request merged**